### PR TITLE
superfile 1.1.0 (new formula)

### PR DESCRIPTION
**Pull request by yorukot**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by chenrui333** [comment]
@MHNightCat there is no need to close the PR and create a new one, that autosquash is a just notification that you need to do the commits squash and forced update your branch would be enough.

File: Formula/s/superfile.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  url "https://github.com/MHNightCat/superfile/archive/refs/tags/v1.1.0.tar.gz"
+  sha256 "703ab37f1bb994e7decc0b48200dd82b24e58ddff8d3204d9cb53fed67ef0f4a"
+  license "MIT"
+  depends_on "go" => :build
+
+  def install
+    Dir.chdir "src" do
+      system "go", "build", "-o", "../bin/spf"
+    end
+    bin.install Dir["bin/*"]
```
Comment (chenrui333) [inline Formula/s/superfile.rb:13]:
```suggestion
    cd "src" do
      system "go", "build", *std_go_args(ldflags: "-s -w", output: bin/"spf")
    end
```

File: Formula/s/superfile.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  end
+
+  test do
+    output = shell_output("#{bin}/spf -v")
+    assert_match("superfile version v1.1.0", output)
```
Comment (chenrui333) [inline Formula/s/superfile.rb:18]:
```suggestion
    # superfile is a GUI application
    assert_match version.to_s, shell_output("#{bin}/spf -v")
```

**Comment by MHNightCat** [comment]
> @MHNightCat there is no need to close the PR and create a new one, that autosquash is a just notification that you need to do the commits squash and forced update your branch would be enough.

Oh sorry i didn't know that

File: Formula/s/superfile.rb [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  homepage "https://github.com/MHNightCat/superfile"
+  url "https://github.com/MHNightCat/superfile/archive/refs/tags/v1.1.0.tar.gz"
+  sha256 "703ab37f1bb994e7decc0b48200dd82b24e58ddff8d3204d9cb53fed67ef0f4a"
+  license "MIT"
+  depends_on "go" => :build
+
+  def install
+    cd "src" do
+      system "go", "build", *std_go_args(ldflags: "-s -w", output: bin/"spf")
+    end
```
Comment (SMillerDev) [inline Formula/s/superfile.rb:12]:
```suggestion
    system "go", "build", *std_go_args(ldflags: "-s -w", output: bin/"spf"), ".../src"
```

Comment (MHNightCat) [reply Formula/s/superfile.rb]: This seems to make it can't to install :(

Comment (chenrui333) [reply Formula/s/superfile.rb:12]: the issue is go.mod is not in the root dir.

**Review by chenrui333 — APPROVED** [review]

File: Formula/s/superfile.rb [hunk 4]
```diff
@@ -0,0 +1,19 @@
+class Superfile < Formula
+  desc "Modern and pretty fancy file manager for the terminal"
+  homepage "https://github.com/MHNightCat/superfile"
+  url "https://github.com/MHNightCat/superfile/archive/refs/tags/v1.1.0.tar.gz"
+  sha256 "703ab37f1bb994e7decc0b48200dd82b24e58ddff8d3204d9cb53fed67ef0f4a"
+  license "MIT"
+  depends_on "go" => :build
```
Comment (chenrui333) [inline Formula/s/superfile.rb:7]:
```suggestion
  license "MIT"

  depends_on "go" => :build
```

**Comment by chenrui333** [comment]
@MHNightCat some final syntax comment, and can you also add the formula into autobump.txt file as well? Thanks!

**Comment by MHNightCat** [comment]
@chenrui333 Sure. thanks for your review!

**Review by chenrui333 — APPROVED** [review]

**Pull request merged**